sympy/vector/tests/test_field_functions.py

```diff
@@ -11,70 +11,78 @@
                                     scalar_potential_difference)
 from sympy.utilities.pytest import raises
 
-C = CoordSysCartesian('C')
-i, j, k = C.base_vectors()
-x, y, z = C.base_scalars()
-delop = C.delop
+cartesian = CoordSystem3D('C')
+
+def _get_spherical_diff_parameters():
+    r, theta, phi = symbols('r, theta, phi')
+    return  Lambda((r, theta, phi), (r * sin(theta) * cos(phi), r * sin(theta) * sin(phi), r * cos(theta)))
+
+spherical = CoordSystem3D('spherical', _get_spherical_diff_parameters())
+
+i, j, k = cartesian.base_vectors()
+x, y, z = cartesian.base_scalars()
+delop = cartesian.delop
 a, b, c, q = symbols('a b c q')
 
+
 def test_del_operator():
 
     #Tests for curl
     assert (delop ^ Vector.zero ==
-            (Derivative(0, C.y) - Derivative(0, C.z))*C.i +
-            (-Derivative(0, C.x) + Derivative(0, C.z))*C.j +
-            (Derivative(0, C.x) - Derivative(0, C.y))*C.k)
+            (Derivative(0, cartesian.y) - Derivative(0, cartesian.z))*cartesian.i +
+            (-Derivative(0, cartesian.x) + Derivative(0, cartesian.z))*cartesian.j +
+            (Derivative(0, cartesian.x) - Derivative(0, cartesian.y))*cartesian.k)
     assert ((delop ^ Vector.zero).doit() == Vector.zero ==
-            curl(Vector.zero, C))
+            curl(Vector.zero, cartesian))
     assert delop.cross(Vector.zero) == delop ^ Vector.zero
     assert (delop ^ i).doit() == Vector.zero
-    assert delop.cross(2*y**2*j, doit = True) == Vector.zero
+    assert delop.cross(2*y**2*j, doit=True) == Vector.zero
```

Comment:
Yes:
``` python
def complex(real, imag=0.0):
    return magic(r=real, i=imag)
```
No:
``` python
def complex(real, imag = 0.0):
    return magic(r = real, i = imag)
```

Copied and pasted from [Other Recommendations](https://www.python.org/dev/peps/pep-0008/#other-recommendations) third bullet point.